support-frontend/assets/components/forms/customFields/sortedOptions.jsx

```diff
@@ -3,13 +3,14 @@
 // ----- Imports ----- //
 
 import React from 'react';
+import { Option } from '@guardian/src-select';
 
 // ----- Functions ----- //
 
 function sortedOptions(os: { [string]: string }): React$Node {
   return Object.keys(os)
     .sort((a, b) => os[a].localeCompare(os[b]))
-    .map(k => <option value={k}>{os[k]}</option>);
+    .map(k => <Option value={k}>{os[k]}</Option>);
```

Comment:
I feel like the variables here could all have clearer names- it took me a moment to figure out what `os` and `k` were referring to (especially since `Option` in other parts of the codebase refers to the Option type!).

Reply:
True. I have some sympathy for it as there are already a lot of things called 'option'! However, I will have a think about how to make it more readable.